src/components/input.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react"
+import styled from "styled-components"
+import colors from "../assets/colors"
+import * as fonts from "../assets/fonts"
+import { block } from "./helpers"
+
+interface InputProps extends React.HTMLProps<HTMLInputElement> {
```

Comment:
Just a thought but the interfaces for the `InputProps` & `TextAreaProps` look pretty similar, well identical at this point. I noticed there is another component for TextBox which I assume will also be similar. Is it worth extracting this out into a reusable Interface. I'm also curious what that implementation would look like. Maybe not something to address in this PR, but I did want to bring it up for dialogue purposes...